### PR TITLE
3567-2

### PR DESCRIPTION
**Pull request by DashaSledopit**
## Номер
3567

## Номер задания
2

## Ссылка на видео с демо
https://www.youtube.com/watch?v=-oDhrZuRzmM

## Комментарии
:)

### CODE REVIEW

File: 3567/2/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (LikaLex) [inline 3567/2/.gitignore]: Empty line.

File: 3567/2/data/camp_numbers.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  3684
+  3587
+  3761
+  3530
+  3561
+  3750
+  3696
+  3567
+  
+
```
Comment (LikaLex) [inline 3567/2/data/camp_numbers.txt]: Useless empty line

File: 3567/2/public/:239975016/checkin/2019-07-23_10:24:43/geolocation.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+53.915452
+27.568695
```
Comment (LikaLex) [inline 3567/2/public/:239975016/checkin/2019-07-23_10:24:43/geolocation.txt]: I think this files shouldn't be there

File: 3567/2/.env [hunk 4]
```diff
@@ -0,0 +1 @@
+TOKEN=961050940:AAGnz79I_lE0plzwTRStgAeN7rqz9X4WGtc
```
Comment (LikaLex) [inline 3567/2/.env]: Why I can see it?

**Comment by DashaSledopit** [comment]
> Still could not run your bot. How you install it?
> Снимок экрана 2019-07-24 в 18 18 19

I think now it will work.

**Pull request merged**